neurom/ezy/neuron.py

```diff
@@ -109,6 +109,8 @@ class Neuron(object):
     def __init__(self, filename, iterable_type=np.array):
         self._iterable_type = iterable_type
         self._nrn = load_neuron(filename, set_tree_type)
+        self.soma = self._nrn.soma
```

Comment:
depending on the type here, I would be concerned that someone might expect to change _nrn, and see show up in soma.  Or is it a container so they point to the same object?